.github/workflows/latexmk.yml

```
@@ -0,0 +1,24 @@
+name: Build LaTeX documents
+on: [push]
+jobs:
+  build_latex:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Compile LaTeX document
+        uses: xu-cheng/latex-action@v2
+        with:
+          root_file: |
+            pleading.tex
+          working_directory:
+            examples
+          latexmk_use_xelatex: true
+      - uses: fregante/setup-git-user@v1
+      - name: Upload PDF to examples branch
+        run: |
+          git fetch origin examples
+          git checkout -b examples origin/examples --merge
+          git add *.pdf
```

```suggestion
          git add *
```
Add all, other files will be ignored anyway.